install/kubernetes/helm/istio/charts/mixer/templates/config.yaml

```diff
@@ -188,8 +190,9 @@ metadata:
   namespace: {{ .Release.Namespace }}
 spec:
   severity: '"Info"'
-  timestamp: request.time
+  timestamp: request.time | timestamp("2017-01-01T00:00:00Z")
   variables:
+    originIp: request.headers["x-forwarded-for"] | "0.0.0.0"
```

Comment:
`x-forwarded-for` can be a list of things right? I feel like we might get into trouble here with `originIp`.  Especially since we default to "0.0.0.0" instead of `ip("0.0.0.0")`.

Should we make this just `xForwardedFor` ?

Reply:
Thanks.. fixed